redbot/core/bank.py

```diff
@@ -312,19 +315,27 @@ def _invalid_amount(amount: int) -> bool:
     ret = []
     accs = await _conf.all_members(guild)
     for user_id, acc in accs.items():
+        member = guild.get_member(user_id)
+        if member is None:
+            member = await bot.get_user_info(user_id)
```

Comment:
isn't doing this counterproductive if this is intended to just get that guild's members?

Wouldn't
```py
if member is None:
    continue
```
be more appropriate here?

Reply:
Given that this is the guild-specific leaderboard, I'm thinking that the way I've done it there makes more sense rather than flat-out skipping over people simply because they happened to not be in the guild at the time the command was run